google-cloud-firestore/lib/google/cloud/firestore/query.rb

```diff
@@ -380,7 +384,63 @@ def limit num
 
           new_query.limit = Google::Protobuf::Int32Value.new value: num
 
-          Query.start new_query, parent_path, client
+          Query.start new_query, parent_path, client, limit_type: :first
+        end
+
+        ##
+        # Limits a query to return only the last matching documents.
+        #
+        # You must specify at least one "order by" clause for limitToLast queries.
+        # (See {#order}.)
+        #
+        # Results for `limit_to_last` queries are only available once all documents
+        # are received. Hence, `limit_to_last` queries cannot be streamed using
+        # {#listen}.
```

Comment:
I am not very familiar with this client and its terminology, but both the Node and the Java client support limit-to-last "listens" via their onSnapshot/Watch API. Not supported are the APIs that stream RunQuery results directly to the user.

Reply:
Thanks for clearing that up. I'll make the change.

Reply:
When you get to this, can you also add a test for Listens with limit-to-last? Thanks

Reply:
Yes, of course!

Reply:
Updated, thank you.